### ISSUE TITLE
Scrolling with mousewheel fails in IE 8

### ISSUE THREAD

**Issue by ghost** [rochal/jQuery-slimScroll#72]
See http://fiddle.jshell.net/p9xMt/show/ for example.

In IE8 you cannot scroll with the mousewheel. I'm not quite sure if you can fix it, but jScrollPane is able to scroll with the mousewheel in IE8.

**Comment by glebmachine** [comment]
Latest version still have same issue(

**Comment by LennyDiggelen** [comment]
yep. same here. Cannot scroll. Only when i select text first in the scrolling element
